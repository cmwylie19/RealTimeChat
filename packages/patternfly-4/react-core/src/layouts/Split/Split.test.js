"use strict";
exports.__esModule = true;
var React = require("react");
var enzyme_1 = require("enzyme");
var Split_1 = require("./Split");
var SplitItem_1 = require("./SplitItem");
var gutters_1 = require("../../styles/gutters");
test('isFilled', function () {
    var view = enzyme_1.mount(<Split_1.Split>
      <SplitItem_1.SplitItem isFilled>Main content</SplitItem_1.SplitItem>
    </Split_1.Split>);
    expect(view).toMatchSnapshot();
});
test('isFilled defaults to false', function () {
    var view = enzyme_1.mount(<Split_1.Split>
      <SplitItem_1.SplitItem>Basic content</SplitItem_1.SplitItem>
    </Split_1.Split>);
    expect(view).toMatchSnapshot();
});
Object.values(gutters_1.GutterSize).forEach(function (gutter) {
    test("Gutter " + gutter, function () {
        var view = enzyme_1.mount(<Split_1.Split gutter={gutter}>
        <SplitItem_1.SplitItem>Basic Content</SplitItem_1.SplitItem>
      </Split_1.Split>);
        expect(view).toMatchSnapshot();
    });
});
