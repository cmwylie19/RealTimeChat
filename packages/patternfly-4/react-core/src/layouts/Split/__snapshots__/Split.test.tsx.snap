// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Gutter lg 1`] = `
<Component
  gutter="lg"
>
  <div
    className="pf-l-split pf-m-gutter"
  >
    <Component>
      <div
        className="pf-l-split__item"
      >
        Basic Content
      </div>
    </Component>
  </div>
</Component>
`;

exports[`Gutter md 1`] = `
<Component
  gutter="md"
>
  <div
    className="pf-l-split pf-m-gutter"
  >
    <Component>
      <div
        className="pf-l-split__item"
      >
        Basic Content
      </div>
    </Component>
  </div>
</Component>
`;

exports[`Gutter sm 1`] = `
<Component
  gutter="sm"
>
  <div
    className="pf-l-split pf-m-gutter"
  >
    <Component>
      <div
        className="pf-l-split__item"
      >
        Basic Content
      </div>
    </Component>
  </div>
</Component>
`;

exports[`isFilled 1`] = `
<Component>
  <div
    className="pf-l-split"
  >
    <Component
      isFilled={true}
    >
      <div
        className="pf-l-split__item pf-m-fill"
      >
        Main content
      </div>
    </Component>
  </div>
</Component>
`;

exports[`isFilled defaults to false 1`] = `
<Component>
  <div
    className="pf-l-split"
  >
    <Component>
      <div
        className="pf-l-split__item"
      >
        Basic content
      </div>
    </Component>
  </div>
</Component>
`;
